src/main/java/seedu/address/logic/commands/CalendarCommand.java

```diff
@@ -0,0 +1,26 @@
+//@@author {ifalluphill}
+
+package seedu.address.logic.commands;
+import seedu.address.commons.core.EventsCenter;
+import seedu.address.commons.events.ui.ShowCalendarRequestEvent;
+
+/**
+ * Opens a calendar window.
+ */
+public class CalendarCommand extends Command {
+
+    public static final String COMMAND_WORD = "calendar";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Opens a calendar window.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String SHOWING_CALENDAR_MESSAGE = "Opened calendar window.";
```

Comment:
Naming isn't consistent with the above.

Reply:
I'm not sure that I understand what you're referring to.

Reply:
Your constants for messages follow the format MESSAGE_X, but your showing calendar message is in the format X_MESSAGE, which would be inconsistent with the naming style used in the project. Code-wise this won't affect anything, but having a consistent naming style is good coding practice.